cmd/transportRequestUploadCTS.go

```diff
@@ -71,6 +73,25 @@ func runTransportRequestUploadCTS(
 
 	log.Entry().Debugf("Entering 'runTransportRequestUpload' with config: %v", config)
 
+	transportRequestID := config.TransportRequestID
+
+	if len(transportRequestID) == 0 {
+
+		from := "origin/master"                         //TODO: make configurable
+		to := "HEAD"                                    //TODO: make configurable
+		transportRequestIdLabel := "TransportRequestID" //TODO: make configurable
```

Comment:
what is "transportRequestIdLabel" for?

Reply:
a typicall commit message in our case looks like:

```
This is the headline

Lorem ispum ...
TransportRequestID: xxxxxxxxxxxx
```

The transportRequestIdLabel is the part before the colon which identifies the line holding the transport request id.

Reply:
shouldn't it be like

```
Lorem ispum ...
TransportRequest: xxxxxxxxxxxx
```

Reply:
In the code above


```
transportRequestIdLabel := "TransportRequestID" //TODO: make configurable
```

 it was "TransportRequestID".

Our default [here](https://github.com/SAP/jenkins-library/blob/master/resources/default_pipeline_environment.yml#L12) (to which we have to stick) is in fact "TransportRequest". So for the real world you are right ...